test/Microsoft.DotNet.ToolPackage.Tests/ToolPackageObtainerTests.cs

```diff
@@ -49,15 +45,16 @@ public void GivenNugetConfigAndPackageNameAndVersionAndTargetFrameworkWhenCallIt
 
             var packageObtainer =
                 ConstructDefaultPackageObtainer(toolsPath);
-            ToolConfigurationAndExecutableDirectory toolConfigurationAndExecutableDirectory =
-                packageObtainer.ObtainAndReturnExecutablePath(
-                    packageId: TestPackageId,
-                    packageVersion: TestPackageVersion,
-                    nugetconfig: nugetConfigPath,
-                    targetframework: _testTargetframework);
 
-            var assetJsonPath = toolConfigurationAndExecutableDirectory
-                .ExecutableDirectory
+            ToolConfigurationAndExecutablePath toolConfigurationAndExecutablePath = packageObtainer.ObtainAndReturnExecutablePath(
+                packageId: TestPackageId,
+                packageVersion: TestPackageVersion,
+                nugetconfig: nugetConfigPath,
+                targetframework: _testTargetframework);
+
+            var assetJsonPath = toolConfigurationAndExecutablePath
+                .Executable
+                .GetDirectoryPath()
                 .GetParentPath()
```

Comment:
If you're going five levels up the directory tree, I'd suggest adding comments noting what each level is.

Reply:
Fixed, added comment